### ISSUE TITLE
Gruntfile not found when dragging project folder into open window

### ISSUE THREAD

**Issue by cperryk** [nickclaw/atom-grunt-runner#25]
I can only get this package to work if I open my project by cd-ing into the project directory and running the "atom" command. If I instead open a new Atom window and drag my project directory into it, I will get a "fatal error: unable to find local grunt" upon attempting to run a grunt command.

**Comment by dwelle** [comment]
`6.1` should solve this, but it's not yet up on apm (#26) so you have to update manually (download zip and replace in your packages folder)

**Comment by cperryk** [comment]
I don't know much about developing these Atom plugins, but here's my theory after fiddling with the code for a few minutes: The plugin's initialization function, which caches the project path, runs when a new window is opened. If the user clicks on a file in the project directory or runs the "atom" command there, a window opens and the path is defined immediately. But when the user opens a blank new window, the path is _not_ defined (how could it be? nothing is open). When the user drags a folder into the window or opens a project via the Project Manager extension, the project path is _then_ defined, but Grunt Runner doesn't know this because it's not listening for the change.

So, basically, to get around this, Grunt Runner either needs to listen for changes to the project path or wait until the user attempts to use Grunt Runner to use the path (because it would be available then). For the latter approach, is it possible the package could just wait until the user tries to use the Run command to parse the gruntfile?

**Comment by ealves-pt** [comment]
Duplicate of #10. Several issues cause ```Error loading Gruntfile: gruntfile not found```
